internal/options/alias.go

```diff
@@ -0,0 +1,246 @@
+package options
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/iancoleman/strcase"
+	"github.com/launchdarkly/ld-find-code-refs/internal/validation"
+	"gopkg.in/yaml.v2"
+)
+
+// We're using multiple packages for configuration.
+// TODO: Use spf13/viper to standardize configuration for args, env, and yaml.
```

Comment:
Because moving everything over to vipe will require a somewhat significant refactor, I haven't spent time writing unit tests for option validation and just tested them manually for the time being